src/common/AdaptiveTree.h

```diff
@@ -29,32 +29,54 @@ namespace adaptive
 template<typename T>
 struct ATTR_DLL_LOCAL SPINCACHE
 {
-  SPINCACHE() :basePos(0) {};
-
-  size_t basePos;
+  /*! \brief Get the <T> value pointer from the specified position
+   *  \param pos The position of <T>
+   *  \return <T> value pointer, otherwise nullptr if not found
+   */
```

Comment:
@glennguy do you think here is appropriate this description? need to be improved in some way?

Reply:
I think it's fine, easy to read